models/users.js

```diff
@@ -41,14 +41,14 @@ const addOrUpdate = async (userData, userId = null) => {
     }
 
     // userId is null, Add or Update user
-    const user = await userModel.where("github_id", "==", userData.github_id).limit(1).get();
-    if (!user.empty) {
-      await userModel.doc(user.docs[0].id).set(userData, { merge: true });
+    const userWithGithubUserId = await userModel.where("github_user_id", "==", userData.github_user_id).limit(1).get();
```

Comment:
NIT: `userWithGithubUserId` can be only`user`

Reply:
Yes, I've changed the variable name in the latest commit. Thank You for suggesting it.